admin_scripts/ubuntu/ubuntu_16.04/xscreensaver/change_time_for_xscreensaver.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+#================================================================
+# HEADER
+#================================================================
+#% SYNOPSIS
+#+    change_time_for_xscreensaver args Time in minutes
+#%
+#% DESCRIPTION
+#%    This changes the text shown for the xscreensaver.
+#%    This script demands that the .xscreensaver file is present in the .skjult folder.
+#%
+#================================================================
+#- IMPLEMENTATION
+#-    version         change_time_for_xscreensaver (magenta.dk) 0.0.1
+#-    author          Danni Als
+#-    copyright       Copyright 2017, Magenta Aps"
+#-    license         GNU General Public License
+#-    email           danni@magenta.dk
+#-
+#================================================================
+#  HISTORY
+#     2017/02/11 : danni : Script creation
+#
+#================================================================
+# END_OF_HEADER
+#================================================================
+
+if [ $# -ne 1 ]
+then
+	echo "This job takes exactly one parameter."
+	exit -1
+fi
+
+TIME=$1
+
+echo "$TIME in minutes"
+
+if [ -z "${TIME##+([0-9])}" ]
+then
+    echo "Input has to be a number."
+    exit -1
+fi
```

Comment:
Do we generally enforce this in scripts as well as in the admin system? `shutdown_and_wakeup.sh`, for example, is perfectly happy to assume that its arguments are integers.